docs/pages/build-reference/limitations.mdx

```diff
@@ -4,46 +4,34 @@ sidebar_title: Limitations
 description: Learn about the current limitations of EAS Build.
 ---
 
-import { Collapsible } from '~/ui/components/Collapsible';
-
 EAS Build is designed to work for any React Native project. However, it is good to be aware of certain limitations that we plan to address since they could prevent you from being able to use the service for your applications or might cause an inconvenience.
 
-## Current limitations
-
-<Collapsible summary="Fixed memory and CPU limits on build worker servers">
+## Fixed memory and CPU limits on build worker servers
 
-The [Server infrastructure reference](/build-reference/infrastructure) contains the most up-to-date information about the current specifications of the Android (Ubuntu) and iOS (macOS) build servers. You may find that the resources available are not sufficient to build your app if your build process requires more than 12GB RAM. In the future, we will add more powerful configurations to increase memory limits and speed up build times.
+The resources available might be insufficient to build your app if your build process requires significant memory. In this case, consider using a `large` resource class](/build-reference/eas-json/#resourceclass-1) in the **eas.json**. See [Android-specific resource class](/build-reference/infrastructure/#android-build-server-configurations) and [iOS-specific resource class](/build-reference/infrastructure/#ios-build-server-configurations).
```

Comment:
Also, I think this PR should be deployed once information about the new `large` resource class  for iOS is added

Reply:
@szdziedzic, is it safe to merge this PR now?

Reply:
Yes, go ahead 😄 Thanks!